problemsets-mat223/modules/module14.tex

```diff
@@ -266,7 +273,15 @@
 \begin{example}
 	Use elementary matrices to find the determinant of $A=\mat{1&2\\3&4}$.
 
-	XXX Finish
+	By applying row reduction to $A$, we could obtain
+	\[
+	    \mat{1&-2\\0&1}\mat{1&0\\0&-\frac{1}{2}}\mat{1&0\\-3&1}\mat{1&2\\3&4} = \mat{1&0\\0&1}.
+	\]
+	Therefore, by the theorem above, we would get
+	\[
+	    1\cdot(-\frac{1}{2})\cdot1\cdot\det(A) = 1,
```

Comment:
Here it should be like `\det(A)=\det(Mat1*mat2*...)=det(mat1)*det(mat2)*...=number`

Reply:
Wow It becomes so hard to compute wwwwww

Reply:
Ohhh actually not